src/Form/MainFormType.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Image;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class MainFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('nom', TextType::class)
+            ->add('url', TextType::class)
```

Comment:
Don't you want to add images using upload instead of asking URLs ?

Reply:
Ok, we'll have to work on it (tryed this morning but didn't work).
Can you validate the rest of the PR so everyone gets it for the holidays? (name fields will be change after we make the form work)